dashboard/config/locales/en.yml

```diff
@@ -1175,6 +1175,9 @@ en:
   return_course_overview: 'Go to course overview'
   view_all_units: 'View all units'
   view_teacher_guide: 'View teacher guide'
+  curriculum_catalog:
+    title: "Curriculum Catalog"
+    description: 'Courses, tutorials, and more designed to teach computer science, artificial intelligence, coding and more to all ages and experience levels.'
```

Comment:
tiniest of nits: missing the oxford comma after "coding" (honestly up to you if you want to update or not)